winui/Rating/ToolTip.md

```diff
@@ -1,12 +1,92 @@
 ---
 layout: post
 title: ToolTip with WinUI Rating control | Syncfusion
-description: Learn here all about Enabletooltip and tooltip format support in the rating items in WinUI rating control.
+description: Learn here all about Enable tooltip features and tooltip format support in the rating items in WinUI rating control.
 platform: WinUI
 control: SfRating
 documentation: ug
 ---
 
 # ToolTip in WinUI Rating
 
-This section explains the tooltip features available in the WinUI Rating control.
+The Rating control EnableToolTip can be customized using the `EnableToolTip` property. The default value is `false`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfRating
+     Value="3"
+     ItemsCount="5"
+     EnableToolTip="True">
+</syncfusion:SfRating>
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfRating rating = new SfRating();
+
+rating.Value = 3;
```

Comment:
@Priyanka-Vijayasankar Why extra space in in C# code snippet.

Reply:
Removed the extra space in C# code snippet.